src/pages/page-my-profile.html

```diff
@@ -565,6 +609,17 @@
                 }
                 this.path = null;
             }
+            /**
+            * blurs the current page
+            */
+            _blur() {
+                if (this.toggleUnlock) {
+                    this.$.container.classList.add('blur');
```

Comment:
I feel this has been added in faffys PR 

          this.customStyle['--my-elem-visibility'] = 'visible';
          this.updateStyles();